generator/src/generator/namespace.rs

```diff
@@ -3224,6 +3232,7 @@ impl<'ns, 'c> NamespaceGenerator<'ns, 'c> {
                     }
                 }
             }
+            outln!(out.indent(), "InvalidValue(u32),");
```

Comment:
Could you add some doc comments to the generated code here? Something explaining that this is not actually part of the protocol, but instead used when the server sends a value that is not understood. It should be treated as a parsing error when encountered, but because there are cases where the server violates the protocol, we do not do so during parsing.

Something like that...

Oh and: Serialising this panics is also worth mentioning.